app/dashboard/static/js/app/view-socs-soc-job-kernel.2017.4.js

```diff
@@ -20,654 +23,483 @@
-                        failCount = labStatus.fail;
-                    } else {
-                        failCount = 0;
-                    }
+        var columns;
 
-                    if (labStatus.hasOwnProperty('pass')) {
-                        passCount = labStatus.pass;
-                    } else {
-                        passCount = 0;
-                    }
+        function _testColumnTitle() {
+            var tooltipNode;
 
-                    if (labStatus.hasOwnProperty('unknown')) {
-                        unknownCount = labStatus.unknown;
-                    } else {
-                        unknownCount = 0;
-                    }
+            tooltipNode = html.tooltip();
+            tooltipNode.setAttribute(
+                'title', 'Total/Successful/Regressions/Other test results');
+            tooltipNode.appendChild(
+                document.createTextNode('Test Results'));
 
-                    totalCount = passCount + failCount + unknownCount;
+            return tooltipNode.outerHTML;
+        }
 
-                    html.replaceContent(
-                        document.getElementById('boot-count-' + lab),
-                        createLabBootCount(
-                            totalCount, passCount, failCount, unknownCount)
-                    );
+        function _renderTestCount(data, type) {
+            return ttest.renderTestCount({data: data, type: type})
+        }
 
-                    html.replaceContent(
-                        document.getElementById('unique-count-' + lab),
-                        createOtherCount(localLab.totals));
-                }
+        function _renderPlanStatus(data, type) {
+            if (type == "display") {
+                var node = document.createElement('div');
+                node.id = "status-" + data;
+                return node.outerHTML;
             }
         }
-    }
 
-    function getBootDoneUnique(response) {
-        var deferred;
+        columns = [
+            {
+                data: 'name',
+                title: 'Test Plan',
+                type: 'string',
+                className: 'test-group-column',
+            },
+            {
+                data: 'name',
+                title: _testColumnTitle(),
+                type: 'string',
+                searchable: false,
+                orderable: false,
+                className: 'test-count pull-center',
+                render: _renderTestCount,
+            },
+            {
+                data: 'name',
+                title: 'Status',
+                type: 'string',
+                searchable: false,
+                orderable: false,
+                className: 'pull-center',
+                render: _renderPlanStatus,
+            },
+        ]
+        gPlansTable
+            .data(results)
+            .columns(columns)
+            .order([0, 'asc'])
+            .rowURL('/soc/%(soc)s/job/%(job)s/kernel/%(kernel)s/plan/%(name)s/')
```

Comment:
The `soc` value is undefined, causing URLs like this one:
```
https://staging.kernelci.org/soc/null/job/kernelci/kernel/staging-20200224.2/plan/baseline/
```
with `/soc/null/`.  Maybe use `gSoC`, or `mach` from the query data?

Reply:
also updated.